pkgs/development/tools/ko/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "ko";
+  version = "0.8.3";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = pname;
+    rev = "v${version}";
+    leaveDotGit = true;
```

Comment:
```suggestion
```

Reply:
This is required for running tests.

Reply:
leaveDotGit ssems to be causing some indeterminism in fetchFromGitHub, I'll find some workaround.

Reply:
> This is required for running tests.

Can we patch the tests instead or just skip the ones relying on it?

Reply:
running `git init` in preCheck works.